libiocage/lib/Distribution.py

```diff
@@ -27,7 +27,7 @@ class DistributionGenerator:
     def __init__(self, host, zfs=None, logger=None):
         self.logger = libiocage.lib.helpers.init_logger(logger)
         self.zfs = libiocage.lib.helpers.init_zfs(zfs)
-        libiocage.lib.helpers.init_host(self, host)
+        self.host = libiocage.lib.helpers.init_host(host, self.logger)
```

Comment:
I prefer to use named arguments - at least for logger and those kind of dependencies. It's much easier to recognize broken interfaces when being explicit.

Reply:
ack, will change